src/index.tsx

```diff
@@ -46,29 +45,37 @@ const history = require('history').createBrowserHistory({
             callback(true);
         }
     },
-    basename: getContextPathBasename()
+    basename: getContextPathBasename(),
 });
 
 const logger = createLogger({
-    collapsed: true
+    collapsed: true,
 });
 
+if (erDev()) {
```

Comment:
```erDev()``` gjelder bare lokal utvikling. Skal vi legge til kjøring av Sentry på q miljøene våre også?

Reply:
Jeg tenker det er flere som tester rare ting i q. (fag og kommune) 
Dessuten har man debug og all info tilgjengelig lokalt. Men når ting feiler i preprod er det ofte vanskeligere å finne ut hvorfor. 

Så jeg tenker ja til q0/q1 :)

Reply:
Jeg har lagt til støtte for q0 og q1 nå også.